src/store/contact-groups/index.ts

```diff
@@ -23,5 +24,12 @@ export const useContactGroupStore = create<ContactGroupsState>()((set, get) => (
 	setStoredOffset: (offset: number): void => set(() => ({ storedOffset: offset })),
 	addStoredContactGroups: (contactGroups: Array<ContactGroup>): void =>
 		set(() => ({ storedContactGroups: [...(get().storedContactGroups ?? []), ...contactGroups] })),
-	emptyStoredContactGroups: (): void => set(() => ({ storedContactGroups: [] }))
+	emptyStoredContactGroups: (): void => set(() => ({ storedContactGroups: [] })),
+	removeStoredContactGroup: (contactGroupId: string): void =>
+		set(() => ({
+			storedContactGroups: get().storedContactGroups.filter(
+				(contactGroup) => contactGroup.id !== contactGroupId
+			),
+			storedOffset: get().storedOffset - 1
```

Comment:
The offset should be updated only if the ID to remove is included in the stored values. It might not be a problem if the remove is used with only valid IDs.